Tools/scripts/deepfreeze.py

```diff
@@ -453,13 +458,10 @@ def main() -> None:
     global verbose
     args = parser.parse_args()
     verbose = args.verbose
-    with open(args.file, encoding="utf-8") as f:
-        source = f.read()
-    modname = args.module or removesuffix(os.path.basename(args.file), ".py")
-    output = args.output or modname + ".c"
-    with open(output, "w", encoding="utf-8") as file:
+    output = args.output or 'deepfreeze.c'
+    with open(output, "w+", encoding="utf-8") as file:
```

Comment:
Why `w+`, why not just `w`? You're not reading the file so no need for `+` right?